packages/dev/loaders/src/glTF/2.0/Extensions/EXT_texture_avif.ts

```diff
@@ -0,0 +1,57 @@
+import type { IGLTFLoaderExtension } from "../glTFLoaderExtension";
+import { GLTFLoader, ArrayItem } from "../glTFLoader";
+import type { ITexture } from "../glTFLoaderInterfaces";
+import type { BaseTexture } from "core/Materials/Textures/baseTexture";
+import type { Nullable } from "core/types";
+import type { IEXTTextureAVIF } from "babylonjs-gltf2interface";
+
+const NAME = "EXT_texture_avif";
+
+/**
+ * [Specification](https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Vendor/EXT_texture_avif/README.md)

```

Comment:
Would be great to drop a comment to say Experimental Support as the spec is still evolving.

Reply:
A link to the PR would be useful as well :-) cause the provided spec link is not available so far

Reply:
Probably should remove the dead link for now. Once the PR is merged, we can update this comment. For now, maybe point to the PR's file for the specification?

https://github.com/KhronosGroup/glTF/blob/5cb7518cf9a1bfb8268320026961b21caf5a4aac/extensions/2.0/Vendor/EXT_texture_avif/README.md